config/routes.rb

```diff
@@ -62,6 +62,10 @@
     resource :locale, only: :update
   end
 
+  namespace :v1, module: "api/v1", as: "api_v1", defaults: { format: "json" } do
```

Comment:
Yes, I added and removed it because I just don't want to deal with setting up the subdomain in the test yet. Will add it back because we want our API doc to show with the subdomain anyway.